### ISSUE TITLE
Unreal Engine 5.5 incompatible Blocks/Airsim on Linux

### ISSUE THREAD

**Issue by Codaero** [Cosys-Lab/Cosys-AirSim#57]
Just recently built Cosys-AirSim from source on both Linux and Windows. I'm able to load the Blocks environment just fine in Windows but on Linux I am getting this error:

**Incompatible or missing module: Blocks
Incompatible or missing module: AirSim**

I've selected the rebuild option for the modules yet it still doesn't work. I'm using the latest Unreal Engine 5.5 and Cosys-AirSim source code.

Attached is my console output from launching my UE5 editor and trying to load the world/rebuild the plugins.

[terminal_output.txt](https://github.com/user-attachments/files/17981409/terminal_output.txt)

**Comment by WouterJansen** [comment]
I have experienced similar issues trying to get it all to work with 5.5 on Linux. I was doing the build on Ubuntu 22.04, but as they have updated clang to 18 I have tried several methods to build the Airlib and other dependencies but still get the same 'undefined symbol' linker issues. Even compiling them with the integrated clang toolchain inside the Unreal Engine installation itself.

 For now I will stick to 5.4 for the supported releases and continue development with 5.5 on Windows.

I might try on Ubuntu 24 where clang18 is more natively supported at a later point. 

TLDR: 5.5 and Cosys-AirSim working fine on Windows (see 5.5 branch), not on Linux.

**Comment by Codaero** [comment]
Tried it and getting a build issue with AnnotationComponent:

[6/82] Compile AnnotationComponent.cpp
/home/codaero/Cosys-AirSim/Unreal/Environments/Blocks/Plugins/AirSim/Source/Annotation/AnnotationComponent.cpp:9:10: fatal error: 'Runtime/Engine/Classes/Materials/Material.h' file not found
    9 | #include "Runtime/Engine/Classes/Materials/Material.h

**Comment by WouterJansen** [comment]
Are you sure you are on the 5.5 branch? The main branch was switched back to 5.4 for stability reasons. 

https://github.com/Cosys-Lab/Cosys-AirSim/blob/0ba18fad015233a18ef9063aa54943932c3b6cad/Unreal/Plugins/AirSim/Source/Annotation/AnnotationComponent.cpp#L9

**Comment by Codaero** [comment]
Sorry, was on the wrong branch. It works now thanks.
